src/vm/prestub.cpp

```diff
@@ -593,6 +597,10 @@ PCODE MethodDesc::MakeJitWorker(COR_ILMETHOD_DECODER* ILHeader, DWORD flags, DWO
                                                 pEntry->m_hrResultCode, 
                                                 TRUE);
                 }
+                else
+                {
+                    g_profControlBlock.pProfInterface->DynamicMethodJITCompilationFinished((FunctionID) this);
```

Comment:
Sorry I didn't catch this before, but the existing JITCompilationFinished has two extra arguments on it that should probably be replicated here. Those values are always constant in the current implementation, but it seems less confusing to match that behavior than to explain why one of them has it and the other doesn't.